src/include/miopen/performance_config.hpp

```diff
@@ -40,7 +40,13 @@ namespace solver {
 
 struct PerfConfig
 {
-    virtual ~PerfConfig() = default;
+    PerfConfig() {}
```

Comment:
Does the absence of this constructor also produce a warning?

Reply:
I am under the impression that simply removing the dtor spec should be sufficient to silence the warning